run_witan_app.sh

```diff
@@ -6,6 +6,8 @@ cat <<EOF  > /root/.witan-app.edn
                      :keyspace "witan"
                      :replication 3}
  :s3 {:bucket "witan-${ENVIRONMENT?not defined}-data"}}
+:kafka {:host "master.mesos"
+         :port 9092}
```

Comment:
I think we might need to use:
- kafka.marathon.mesos for the broker(s)
- master.mesos for the zookeeper instance